taylor-marc/styles/layout.css

```diff
@@ -0,0 +1,79 @@
+body > header {
+  background-color: #ffeaa9;
+  position: relative;
+  padding: 10px;
+}
+
+.hamburger {
```

Comment:
I would probably put the hamburger styles in modules because hamburger menus tend to be a very specific one time thing, but I could see there being a case to put it in layout.